starter-code/iteration-1/main.ts

```diff
@@ -1,18 +1,42 @@
 // Iteration 1
```

Comment:
The solution works but try packing everything in the class since we are practicing TypeScript and ES6. So you would get something like this: